<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableDesign</class>
 <widget class="QWidget" name="TableDesign">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>字段名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="typeButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Integer</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Varchar</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Float</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Boolean</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Date</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="typeLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>字段类型：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lenLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>字段长度：</string>
   </property>
  </widget>
  <widget class="QLabel" name="namelabel_4">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>70</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lenEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="defaultLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>默认值：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="defaultEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="minnLabel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>70</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>最小值：</string>
   </property>
  </widget>
  <widget class="QLabel" name="maxnLabel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>110</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>最大值：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="minnEdit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>70</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="maxnEdit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>110</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="pkCheck">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>150</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>主键</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="unCheck">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>150</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>唯一键</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="nonCheck">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>190</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>非空</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="comEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>230</y>
     <width>391</width>
     <height>87</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="comLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>字段注释：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>360</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>360</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>重置</string>
   </property>
  </widget>
  <widget class="QLabel" name="namelabel_9">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>插入字段</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>typeButton</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>TableDesign</receiver>
   <slot>on_comboBox_CIC(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>35</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>TableDesign</receiver>
   <slot>on_button_confirm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>369</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>376</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>TableDesign</receiver>
   <slot>on_button_reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>370</y>
    </hint>
    <hint type="destinationlabel">
     <x>481</x>
     <y>395</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pkCheck</sender>
   <signal>clicked()</signal>
   <receiver>TableDesign</receiver>
   <slot>on_checkBox_pkc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>428</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_comboBox_CIC(QString)</slot>
  <slot>on_button_confirm()</slot>
  <slot>on_checkBox_pkc()</slot>
  <slot>on_button_reset()</slot>
 </slots>
</ui>
